core/src/block.rs

```diff
@@ -876,17 +876,11 @@ pub mod stream {
     }
 
     /// Message sent by the stream producer
+    /// Block sent by the peer.
     #[version_with_scale(n = 1, versioned = "VersionedBlockPublisherMessage")]
-    #[derive(Debug, Clone, Decode, Encode, FromVariant, IntoSchema)]
+    #[derive(Debug, Clone, Decode, Encode, IntoSchema)]
     #[allow(clippy::large_enum_variant)]
-    pub enum BlockPublisherMessage {
-        /// Answer sent by the peer.
-        /// The message means that block stream connection is initialized and will be supplying
-        /// events starting with the next message.
-        SubscriptionAccepted,
-        /// Block sent by the peer.
-        Block(VersionedCommittedBlock),
-    }
+    pub struct BlockPublisherMessage(pub VersionedCommittedBlock);
```

Comment:
consider renaming to `EventMessage` or `Event`

Reply:
i will stick with `BlockMessage`